src/read.jl

```diff
@@ -104,113 +104,160 @@ function read_storage(filepath)::Storage
 end
 
 
-"""Load demand flexibility profile from .csv files into DataFrame(s)."""
+"""
+    read_demand_flexibility(filepath, interval)
+
+Load demand flexibility profiles and parameters from .csv files and return them in a 
+DemandFlexibility struct.
```

Comment:
I see the convention. Should we have an indent here?

Reply:
I was going off of this: https://github.com/invenia/BlueStyle#documentation

Reply:
It looks like the function call example should be indented, but the other documentation should not: https://docs.julialang.org/en/v1/manual/documentation/

Reply:
I'm just comparing this with l.87 and l.45.

Reply:
Aha, l.45 and l.87 of *model.jl*. I think in this case, both other style guides agree and so I think our style should be updated.

Reply:
Ah, thanks for finding that Daniel, I wasn't sure what l.87 and l.45 were. I think as Daniel mentioned in a recent PR, we will probably need a more extensive PR at some point to refactor the `REISE.jl` code to be compliant with these style guides.

Reply:
Would it be helpful for me to create an Issue focusing on refactoring the `REISE.jl` code to follow one of the other code styles? I know we discussed `Blue` in the past

Reply:
Probably.